packages/apidom-ls/package.json

```
@@ -20,9 +20,61 @@
   "unpkg": "./dist/apidom-ls.browser.min.js",
   "main": "./cjs/index.cjs",
   "exports": {
-    "types": "./types/dist.d.ts",
-    "import": "./es/index.js",
-    "require": "./cjs/index.cjs"
+    ".": {
+      "types": "./types/dist.d.ts",
+      "import": "./es/index.js",
+      "require": "./cjs/index.cjs"
+    },
+    "./services/validation/providers/json-schema-validation-provider": {
+      "import": "./es/services/validation/providers/json-schema-validation-provider.js",
+      "require": "./cjs/services/validation/providers/json-schema-validation-provider.cjs",
+      "types": "./types/dist.d.ts"
```

Types must be referenced as individual typing files, not as typing bundle.

```suggestion
      "types": "./types/services/validation/providers/json-schema-validation-provider.d.ts"
```

In order to make this possible this line https://github.com/swagger-api/apidom/blob/1808ea06d75ca5e5c40ddd5719bd01da48fd3758/packages/apidom-ls/package.json#L79 needs to be changed to:

```json
 "types/",
```